src/lib.rs

```diff
@@ -183,7 +190,7 @@ mod file_error;
 mod key_file;
 pub mod prelude;
 pub mod signal;
-mod source;
+pub mod source;
```

Comment:
Part of source reexported, while part is not.
Not better do `pub use source::*;` ?